MonoGame.Framework/Game.cs

```diff
@@ -404,6 +406,13 @@ public void Run(GameRunBehavior runBehavior)
             }
         }
 
+        internal void ExitEverything()

```

Comment:
If we're gonna do this it should at least be documented why it exists and wrapped with `#if DESKTOPGL` to avoid people from thinking it is how things should get shut down.